src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1991,15 +1990,14 @@ internal IOperation CreateBoundTupleLiteralOperation(BoundTupleLiteral boundTupl
 
         internal IOperation CreateBoundConvertedTupleLiteralOperation(BoundConvertedTupleLiteral boundConvertedTupleLiteral, bool createDeclaration = true)
         {
-            return CreateTupleOperation(boundConvertedTupleLiteral, boundConvertedTupleLiteral.SourceTuple.Type, createDeclaration);
+            return CreateTupleOperation(boundConvertedTupleLiteral, boundConvertedTupleLiteral.SourceTuple?.Type, createDeclaration);
```

Comment:
This is the first actual bug nullable-related bug I've found as part of this porting effort. I'm honestly quite impressed.